src/components/section/section.md

```diff
@@ -0,0 +1,41 @@
+Sections can be used to semantically group controls, information, and inputs. Sections get the `region` ARIA role by default, and the `title` prop is required to support accessibility with a contained heading. For now, the section component should only be used in Custom Apps, while the "page layout" spec is still in flux.
+
+```
+<Section
+  title="I am a section"
+  description="Test description"
+/>
+```
+
+Sections can contain child elements, like any other element:
+```
+<Section
+  title="Project Details"
+  description="These fields allow control of custom project fields."
+>
+<CustomFieldInputText
```

Comment:
Shouldn't nested elements be indented?

Reply:
Yupp, I'll do this. Damn examples for not failing linting! 😉